pkg/utils/k8sutil.go

```diff
@@ -53,7 +53,8 @@ import (
 
 const (
 	controllerImage = "bitnami/kubeless-controller"
-	kafkaImage      = "wurstmeister/kafka"
+	kafkaImage      = "bitnami/kafka"
```

Comment:
ideally there should be a default which would be the released tag (reference by a sha256) and the image could be overwritten by an ENV variabale. That way you can set the local env VAR to bitnami/kafka:latest for testing, but the release defaults to a specific well tested version. Helm does this.